Packs/Core/Playbooks/playbook-Exchange_forwarding_rule_configured.yml

```
@@ -0,0 +1,1575 @@
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": -410,
+          "y": 660
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 150bae48-03a8-495a-87b5-11b63bd85444
+    type: regular
+    task:
+      id: 150bae48-03a8-495a-87b5-11b63bd85444
+      version: -1
+      name: Get user risk score
+      description: Retrieve the risk score of a specific user or list of users with the highest risk score in the environment along with the reason affecting each score.
+      script: '|||core-list-risky-users'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "21"
+    scriptarguments:
+      user_id:
+        simple: ${Core.OriginalAlert.raw_abioc.event.identity_normalized.identity}
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 840,
+          "y": 660
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 146faa60-9405-4cf2-8f7c-7ce02160a0c4
+    type: regular
+    task:
+      id: 146faa60-9405-4cf2-8f7c-7ce02160a0c4
+      version: -1
+      name: Check if the rule was created outside of business hours
+      description: Checks whether the given value is within the specified time (hour) range.
+      scriptName: BetweenHours
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "23"
+    scriptarguments:
+      begin_time:
+        simple: "22:00:00"
+      end_time:
+        simple: "06:00:00"
+      value:
+        complex:
+          root: Core.OriginalAlert.event
+          accessor: event_timestamp
+          transformers:
+          - operator: TimeStampToDate
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 60,
+          "y": 670
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: cdba5566-f4de-4815-85ba-46d04083adf2
+    type: regular
+    task:
+      id: cdba5566-f4de-4815-85ba-46d04083adf2
+      version: -1
+      name: Assess geolocation anomalies
+      description: |-
+        Returns all elements from the left side that have a substring that is equal to an element from the right side. Note: This filter is case-insensitive.
+        E.g -AnyMatch left=baby right=A will return baby. For more examples see the filter's Readme.
```

```suggestion
        E.g., AnyMatch left=baby right=A will return baby. For more examples see the filter's Readme.
```
Where is the filter's Readme? Either add a link, or remove.